export const signUp = async (req: Request, res: Response) => {
  try {
  } catch (error) {}
};
